src/earn/EarnPoolInfoScreen.tsx

```diff
@@ -107,14 +110,156 @@ function TokenIcons({
+          {earnItem.label}
+        </Text>
+      </View>
+      <View style={styles.flexShrink}>
+        <Text style={styles.depositAndEarningsCardValueText}>
+          {t('earnFlow.poolInfoScreen.lineItemAmountDisplay', {
+            localCurrencySymbol,
+            localCurrencyAmount: formatValueToDisplay(amountInLocalCurrency),
+            cryptoAmount: formatValueToDisplay(new BigNumber(earnItem.amount)),
+            cryptoSymbol: tokenInfo?.symbol,
+          })}
+        </Text>
+      </View>
+    </View>
+  )
+}
+
+function DepositAndEarningsCard({ earnPosition }: { earnPosition: EarnPosition }) {
+  const { t } = useTranslation()
+  const { balance } = earnPosition
+  const { earningItems, depositTokenId, cantSeparateCompoundedInterest } = earnPosition.dataProps
+  const tokenInfo = useTokenInfo(depositTokenId)
+  const localCurrencySymbol = useSelector(getLocalCurrencySymbol)
+  const localCurrencyExchangeRate = useSelector(usdToLocalCurrencyRateSelector)
+
+  // Deposit items used to calculate the total balance and total deposited
+  const depositBalanceInUsd = tokenInfo?.priceUsd?.multipliedBy(balance)
+  const depositBalanceInLocalCurrency = new BigNumber(localCurrencyExchangeRate ?? 0).multipliedBy(
+    depositBalanceInUsd ?? 0
+  )
+
+  // Earning items used to calculate the total balance and total deposited
+  const earningItemsTokenIds = earningItems.map((item) => item.tokenId)
+  const earningItemsTokenInfo = useTokensInfo(earningItemsTokenIds)
+
+  const totalBalanceInLocalCurrency = depositBalanceInLocalCurrency.plus(
+    earningItems.reduce((acc, item) => {
+      if (item.includedInPoolBalance) return acc
+      const tokenInfo = earningItemsTokenInfo.find((token) => token?.tokenId === item.tokenId)
+      const amountInUsd = tokenInfo?.priceUsd?.multipliedBy(item.amount)
+      const amountInLocalCurrency = new BigNumber(localCurrencyExchangeRate ?? 0).multipliedBy(
+        amountInUsd ?? 0
+      )
+      return acc.plus(amountInLocalCurrency ?? 0)
+    }, new BigNumber(0))
+  )
+
+  const totalDepositBalanceInLocalCurrency = depositBalanceInLocalCurrency.minus(
+    earningItems.reduce((acc, item) => {
+      if (!item.includedInPoolBalance) return acc
+      const tokenInfo = earningItemsTokenInfo.find((token) => token?.tokenId === item.tokenId)
+      const amountInUsd = tokenInfo?.priceUsd?.multipliedBy(item.amount)
+      const amountInLocalCurrency = new BigNumber(localCurrencyExchangeRate ?? 0).multipliedBy(
+        amountInUsd ?? 0
+      )
+      return acc.plus(amountInLocalCurrency ?? 0)
+    }, new BigNumber(0))
+  )
+
+  const totalDepositBalanceInCrypto = new BigNumber(balance).minus(
+    earningItems.reduce((acc, item) => {
+      // Don't include earning items that are not part of the pool balance or are not the deposit token
+      if (!item.includedInPoolBalance || item.tokenId !== depositTokenId) return acc
+      return acc.plus(item.amount)
+    }, new BigNumber(0))
+  )
```

Comment:
```suggestion
  const totalDepositBalanceInCrypto = totalDepositBalanceInLocalCurrency.dividedBy(new BigNumber(localCurrencyExchangeRate ?? 1)).dividedBy(new BigNumber(tokenInfo?.priceUsd ?? 1))
```
I think what you have now would cause an issue if `includedInPoolBalance` is true but `item.tokenId !== depositTokenId`. In that case, we wouldn't subtract it, but we would subtract it for the fiat calculation in `totalDepositBalanceInLocalCurrency`, so the line item would show `fiat (crypto)` but the crypto amount would actually be more than the fiat amount converts to. 

For example, if there is an earningItem where token is USDT and balance is 5, and then the poolBalance is 25 and token is USDC, for the fiat calculation (`totalDepositBalanceInLocalCurrency`) we would do $25 - $5 = $20, but for crypto, since tokens don't match we wouldn't subtract so `totalDepositBalanceInCrypto` would be set to 25. Then for the line item we would show `$20 (25 USDC)`.

I added a suggestion here of what we could do instead.

Reply:
Fixed in 970dd5de7488c541c35d88fca14fb4aa507626db!